src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -16,7 +19,7 @@ public abstract class DownloadCommandBase : Command
         private readonly IPackageRepository _cacheRepository;
         private readonly List<string> _sources = new List<string>();
 
-        protected PackageSaveModes EffectivePackageSaveMode { get; set; }
+        protected PackageSaveMode EffectivePackageSaveMode { get; set; }
```

Comment:
Set it to a package extraction default value and skip redundant (if == None) statements later